### PR TITLE
Added unit tests for models endpoints

### PR DESCRIPTION
**Pull request by HermanPlay**
Do we also want to include model_details tests in this module, or should it be done in a separate module?

### CODE REVIEW

File: app/api/tests/test_model.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    response = client.put(MODEL_ROUTE, json=dict(test_model))
+    model_id = response.json()["id"]
+
+    test_model_patch = model_schemas.PatchModel(
+        name="test_patch_name",
+        description="test_patch_description",
+        status=model_schemas.Status.INACTIVE,
+        updated_by=user_id_second,
+    )
+    excepted_output = dict(test_model_patch)
```
Comment (TheRealSeber) [inline app/api/tests/test_model.py:64]: bro expected

File: app/api/tests/test_model.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+    test_model = model_schemas.PutModel(
+        name="test_name_2",
+        description="test_description_2",
+        status=model_schemas.Status.INACTIVE,
+        created_by=user_id,
+    )
+    response_model_2 = client.put(MODEL_ROUTE, json=dict(test_model))
+
+    expected_responce = [response_model_1.json(), response_model_2.json()]
```
Comment (TheRealSeber) [inline app/api/tests/test_model.py:115]: repsonse bro

**Review by TheRealSeber — COMMENTED** [review]
check users also for misspellings

**Comment by matined** [comment]
I'd put model_details in a separate file just to make files shorter and more readable

**Review by matined — APPROVED** [review]

**Pull request merged**